src/components/RegisterForm/RegisterForm.jsx

```diff
@@ -0,0 +1,116 @@
+    >
+      {(formik) => (
+        <div className="w-full max-w-xs mx-auto">
+          <form
+            onSubmit={formik.handleSubmit}
+            className="bg-white shadow-md  rounded px-8 pt-6 pb-8 mb-4"
+          >
+            <label
+              htmlFor="name"
+              className="block text-gray-700 text-sm font-bold mb-2"
+            >
+              {t('Name')}
+              <input
+                className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
+                id="name"
+                name="name"
+                type="text"
+                placeholder="Enter your Name"
+                onChange={formik.handleChange}
+              />
+            </label>
+            {formik.errors.name && (
+              <div className="text-red-500 text-sm font-bold">
+                {formik.errors.name}
+              </div>
+            )}
+            <label
+              htmlFor="email"
+              className="block text-gray-700 text-sm font-bold mb-2"
+            >
+              {t('Email')}
+              <input
+                className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
+                id="email"
+                name="email"
+                type="text"
+                placeholder="Enter your email"
+                onChange={formik.handleChange}
+              />
+            </label>
+            {formik.errors.email && (
+              <div className="text-red-500 text-sm font-bold">
+                {formik.errors.email}
+              </div>
+            )}
+            <label
+              htmlFor="password"
+              className="block text-gray-700 text-sm font-bold mb-2"
+            >
+              {t('Password')}
+              <input
+                className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
+                id="password"
+                name="password"
+                type="password"
+                placeholder="Enter your password"
+                onChange={formik.handleChange}
+              />
+            </label>
+            {formik.errors.password && (
+              <div className="text-red-500 text-sm font-bold">
+                {formik.errors.password}
+              </div>
+            )}
+            <label
+              htmlFor="confirmPassword"
+              className="block text-gray-700 text-sm font-bold mb-2"
+            >
+              {t('confirmPassword')}
+              <input
+                className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
+                id="confirmPassword"
+                name="confirmPassword"
+                type="password"
+                placeholder="Confirm your password"
```

Comment:
This should be a t('')

Reply:
@MuhannedNoman  @DhiaaAlshamy 
should I make  t in name and id  also ????? or only  placeholder

Reply:
@Marwa-k  make it for the text that appears on the screen < which is the placeholder